boards/frdm_k64f.overlay

```diff
@@ -1,13 +1,8 @@
 &flash0 {
 	partitions {
-		app_state_partition: partition@10000 {
-			label = "application-state";
-			reg = <0x00010000 0x0000f000>;
-		};
-
-		credentials_partition: partition@1f000 {
+		credentials_partition: partition@10000 {
```

Comment:
NOTE: Here, I moved the credentials partition out of the way of the storage partition on K64F.